### ISSUE TITLE
how to train the model on 80G A100 gpus？

### ISSUE THREAD

**Issue by jiujing23333** [hpcaitech/Open-Sora#844]
Can I increase the tp_size or pp_size in the configuration to adapt to lower GPU memory usage or support higher-resolution video training? Have you tried mixing multiple parallelization strategies, and could there be any potential bugs?
Thanks

**Comment by SimonWXW** [comment]
Yes, you can try increasing the tp_size or pp_size. Also, mixing multiple parallelization strategies is an option and may provide benefits, though it might also introduce some potential bugs. If you run into any issues or unexpected behavior, please feel free to leave another comment so we can help troubleshoot.

**Comment by MukundVarmaT** [comment]
Hi were you able to finetune on 80G cards? I am interested to fully finetune the model. Wanted to ask about GPU requirements, and training time. 

Thanks

**Comment by botbw** [comment]
@jiujing23333 @MukundVarmaT 

Sure, you can try increasing `tp_size` and `pp_size` to reduce memory consumption, but please take note that we didn't test all possible parallel strategies and there could be possible failure (especially when tensor dim cannot be fully divided/partitioned).

Regarding fine-tuning/training memory usage, you can refer to the 8-GPU statistics [here](https://github.com/hpcaitech/Open-Sora/issues/823#issuecomment-2764448134)
